packages/react-test-renderer/src/ReactShallowRenderer.js

```diff
@@ -147,9 +147,11 @@ class ReactShallowRenderer {
     // Fallback to previous instance state to support rendering React.cloneElement()
     const state = this._newState || this._instance.state || emptyObject;
 
-    if (typeof this._instance.shouldComponentUpdate === 'function') {
+    if (
+      typeof this._instance.shouldComponentUpdate === 'function' &&
+      this._forcedUpdate !== true
```

Comment:
This can just be `!this._forcedUpdate`, right?

Reply:
Yes, `_forcedUpdate` is always a boolean